pkg/features/kube_features.go

```diff
@@ -116,6 +116,12 @@ const (
 	// Add priority to pods. Priority affects scheduling and preemption of pods.
 	PodPriority utilfeature.Feature = "PodPriority"
 
+	// owner: @bsalamat
+	// alpha: v1.8
+	//
+	// Allow preemption of pods with lower priority. Only work with PodPriority is enabled.
+	PodPreemption utilfeature.Feature = "PodPreemption"
```

Comment:
Enabling preemption without enabling priority is wrong configuration. I think we should check and throw error if it happens. I don't know if we have any standard location in our codebase to check for such dependencies of feature gates.

Reply:
Thanks, I will make it a scheduler configuration.